src/Bundle/DependencyInjection/TalleuRedisOmExtension.php

```diff
@@ -45,7 +45,7 @@ private function registerApiPlatformServices(ContainerBuilder $container)
         $this->registerFilters($container);
     }
 
-    private function registerProviders(ContainerBuilder $container)
+    private function registerProviders(ContainerBuilder $container): void
```

Comment:
All of this could be done directly in the XML file.
Also, for performance and reliability reasons, you shouldn't use autowiring and autoconfiguration in bundles.